File: app/src/main/java/com/example/tutorial/db/NotesDbHelper.java
package com.example.tutorial.db;

import android.content.Context;
import android.database.sqlite.SQLiteDatabase;
import android.database.sqlite.SQLiteOpenHelper;

import androidx.annotation.Nullable;

public class NotesDbHelper extends SQLiteOpenHelper {
    public NotesDbHelper(@Nullable Context context) {
        super(context, NotesContract.DB_NAME, null, NotesContract.DB_VERSION);
    }

    @Override
    public void onCreate(SQLiteDatabase db) {
        for (String query : NotesContract.CREATE_DATABASE_QUERIES) {
            db.execSQL(query);
        }
    }

    @Override
    public void onUpgrade(SQLiteDatabase db, int oldVersion, int newVersion) {

    }
}

File: app/src/main/java/com/example/tutorial/db/NotesDbHelper.kt
package com.example.tutorial.db

import android.content.Context
import android.database.sqlite.SQLiteOpenHelper
import android.database.sqlite.SQLiteDatabase

class NotesDbHelper(context: Context?) : SQLiteOpenHelper(context, NotesContract.DB_NAME, null, NotesContract.DB_VERSION) {

    override fun onCreate(db: SQLiteDatabase) {
        for (query in NotesContract.CREATE_DATABASE_QUERIES) {
            db.execSQL(query)
        }
    }

    override fun onUpgrade(db: SQLiteDatabase, oldVersion: Int, newVersion: Int) {}
}